doc/manuals/user/pagination.md

```diff
@@ -80,11 +135,15 @@ POST <orion_host>:1026/v1/queryContext?offset=1000&limit=100
 
 ## Ordering results
 
-However, in the case of entities query, the `orderBy` URL parameter can be use to
-specify a different order. In particular, the value of `orderBy` is a comma-separated
-list of attributes and the results are ordered by the first attribute. On ties,
-the results are ordered by the second attribute and so on.
-A "!" before the attribute name means that the order is reversed. For example:
+In the case of entities query, the `orderBy` URL parameter can be use to
+specify an order different from creation time. In particular, the value of
+`orderBy` is a comma-separated list of attributes and the results are ordered
+by the first attribute. On ties, the results are ordered by the second attribute and so on.
+A "!" before the attribute name means that the order is reversed. For example (NGSIv2):
+
```

Comment:
I mean that if a new entity is created between pagination 1 and 2, an entity might be lost as the order has changed due to the new entity.

When the order is on creation-time, any new entity is always placed last in the sort, thus never 'destroying' the list